play-json/shared/src/main/scala/play/api/libs/json/JsLookup.scala

```diff
@@ -86,7 +86,7 @@ case class JsLookup(result: JsLookupResult) extends AnyVal {
    */
   def \(index: Int): JsLookupResult = result match {
     case JsDefined(arr: JsArray) =>
-      arr.value.lift(index).map(JsDefined.apply).getOrElse(JsUndefined(s"Array index out of bounds in $arr"))
+      arr.value.lift(index).map(JsDefined.apply _).getOrElse(JsUndefined(s"Array index out of bounds in $arr"))
```

Comment:
is this change really needed? I reverted it and see no warnings...

Reply:
Nice find!  Yeah, I see the same.  My guess is these came up while typechecking was in a failing state from other, legit errors...? 🤷🏼‍♂️